CHANGELOG.md

```diff
@@ -2,6 +2,162 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.1.0-rc.21](https://github.com/ironSource/fusion-ui/compare/v1.1.0-rc.20...v1.1.0-rc.21) (2022-02-23)
```

Comment:
remove all rc.X mentions. instead add everything under version 1.1.0

Reply:
will do it after or right before the release build I think.